source/our_tools/recobundles.rst

```diff
@@ -0,0 +1,95 @@
+RecobundlesX
+============
+
+Here is a typical example of a script to run our scilpy's version of Recobundles on a batch of subjects. Dipy also has its version, and you can find the explanation `here <https://dipy.org/documentation/0.16.0./examples_built/bundle_extraction/>`_.
```

Comment:
Can we add a line here to say that dipy's version is single atlas, single parameter.
Our version, is multi-atlas and multi-parameter, and as shown in Rheault PhD 2020, is more robust.

Moreover, the user does not need to make unecessary copy of the whole-brain tractogram while being registered.

@frheault help please

Reply:
@mdesco , modification is ok for you?